models.yaml

```diff
@@ -0,0 +1,254 @@
+---
+models:
+- api_name: gpt-4o
+  name: GPT-4o (latest)
+  supports_images: true
+  supports_tools: true
+  input_token_cost_cents: '0.00025'
+  output_token_cost_cents: '0.001'
```

Comment:
Just like with strings, you don't need to put the floats inside quotes. But it's also okay to leave them.